src/System.Windows.Extensions/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
It's currently a non-goal to expose this library to netstandard since it isn't portable.  I would assert that the purpose of this assembly is to add a bunch of Windows-specific types back to netcoreapp.  Given they aren't even available on Linux netcoreapp I wouldn't want to suggest they'd be available to all netstandard frameworks.